mw-webapp/src/service/WayService.ts

```diff
@@ -118,4 +125,24 @@ export class WayService {
     return validatedFavoriteWaysDTO;
   }
 
+  /**
+   * Update favoriteWayUuids of UserDto and favoriteForUserUuids of WayDto
+   */
+  public static async updateFavoriteForUserUuidsWithBatch(
+    userUuid: string,
+    wayUuid: string,
+    updatedFavoriteForUserUuids: string[],
+    updatedFavoriteWays: string[],
+  ) {
+    const batch = writeBatch(db);
```

Comment:
It's good, but let's move batching in DAL layer. Let's in the service layer  will be only batch.update and similar methods. I think it will be better  now))

Reply:
Done.